edenai_apis/apis/google/google_helpers.py

```
@@ -109,6 +109,52 @@ def handle_google_call(function_to_call, **kwargs):
+
+    try:
+        response = await function_to_call(**kwargs)
+    except GoogleAPIError as exc:
+        try:
+            status_code = exc.code
+            if isinstance(status_code, HTTPStatus):
+                status_code = status_code.value
+            if not isinstance(status_code, int):
+                try:
+                    status_code = int(status_code)
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)